sql/dbupdate.php

```diff
@@ -480,4 +480,36 @@
     ]);
 }
 ?>
+<#42>
+<?php
+// Introducing PublicationUsageGroup for grouping PublicationUsage.
+\srag\Plugins\Opencast\Model\Publication\Config\PublicationUsageGroup::updateDB();
```

Comment:
We should already start to introduce new tables with ilDBInterface explicitly and not via the ActiveRecord-function updateDB. ActiveRecord will probably be discontinued at some point with ILIAS 11 or so. updateDB also has the problem that adjustments to a table can no longer be traced later because the "current" version of the table is always installed the first time it is called up.